### PR TITLE
Leadership arabic styles

### PR DESCRIPTION
**Pull request by padmashreeshetty123**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--world-bank--aemsites.aem.live/ext/ar/leadership
- After: https://leadership-arabic-styles--world-bank--aemsites.aem.live/ext/ar/leadership

### CODE REVIEW

File: blocks/heading/heading.css [hunk 1]
```diff
@@ -18,6 +18,11 @@
     line-height: 31.36px;
 }
 
+html[dir="rtl"] .heading-wrapper .heading h2{
+text-align: right;
```
Comment (thomsebastin) [inline blocks/heading/heading.css:22]: I think if you use `text-align: end`, it'll work for both rtl and non-rtl without any extra styles. Can you try?

Comment (padmashreeshetty123) [reply blocks/heading/heading.css:22]: sure

**Comment by mkbansal1** [comment]
@padmashreeshetty123  please resolve conflicts

File: blocks/cards/cards.css [hunk 2]
```diff
@@ -66,7 +66,10 @@
   margin-top: var(--spacing-large);
 }
 
-.cards.profile-summary .cards-card-body h1,h2,h3,h4 {
+.cards.profile-summary .cards-card-body h1,
+h2,
+h3,
+h4 {
   font-size: var(--font-size-1100);
```
Comment (mkbansal1) [inline blocks/cards/cards.css:73]: Rules for h2, h3, h4 are open ended and should be covered by context of component
.cards.profile-summary .cards-card-body h1,
.cards.profile-summary .cards-card-body h2,
.cards.profile-summary .cards-card-body h3,
.cards.profile-summary .cards-card-body h4

**Review by mkbansal1 — CHANGES_REQUESTED** [review]

File: blocks/cards/cards.css (hunk 2 above)
Comment (padmashreeshetty123) [reply blocks/cards/cards.css:73]: okay

**Review by mkbansal1 — APPROVED** [review]

**Pull request merged**